bioio_ome_tiled_tiff/reader.py

```diff
@@ -44,66 +63,184 @@ class Reader(BaseReader):
     _fs: "AbstractFileSystem"
     _path: str
 
-    # Required Methods
+    backend: Optional[str] = None
```

Comment:
Doesn't seem like this is ever set. Based on the `is_supported_image` method I wonder if this should just always be `python` anyway.

Reply:
resolved in [3e46353](https://github.com/bioio-devs/bioio-ome-tiled-tiff/pull/2/commits/3e46353b361a2b17d4edfec8399cafba3914344b).

Reply:
It is true you now set it to "python" but it is still unused and never changes. Seems like it should either be a param to the constructor or removed entirely

Reply:
cool yeah ill remove it